pkg/providers/cloudstack/cloudstack.go

```diff
@@ -590,32 +595,48 @@ func AnyImmutableFieldChanged(oldCsdc, newCsdc *v1alpha1.CloudStackDatacenterCon
 			oldAz.Account == newAz.Account &&
 			oldAz.Domain == newAz.Domain) {
 
+			log.Info("Old and new CloudStackDatacenterConfigs do not match", "oldAvailabilityZone", oldAz,
+				"newAvailabilityZone", newAz)
 			return true
 		}
 	}
 
 	if !oldCsmc.Spec.Template.Equal(&newCsmc.Spec.Template) {
+		log.Info("Old and new CloudStackMachineConfig Templates do not match", "machineConfig", oldCsmc.Name, "oldTemplate", oldCsmc.Spec.Template,
```

Comment:
```suggestion
		log.V(4).Info("Old and new CloudStackMachineConfig Templates do not match", "machineConfig", oldCsmc.Name, "oldTemplate", oldCsmc.Spec.Template,
```

Same for the other new logs similar to this one
For reference: https://github.com/aws/eks-anywhere/blob/main/pkg/logger/doc.go#L17

Reply:
I created an experiment for testing this logging configuration. I added a couple of test logs in this function:

```
	log.V(4).Info("[DEBUG] Testing logging capabilities (v4)")
	log.Info("[DEBUG] Testing logging capabilities (no verbosity)")
```

and then replaced the eks-a controller in my cluster. I observed the no verbosity logs many times, but didn't see any instances of V4 logs

```
1.664898488914078e+09	INFO	controllers.Cluster	[DEBUG] Testing logging capabilities (no verbosity)
```

Reply:
That's probably because the logger in the controller is not initialized to output v4 logs
We can change that if we want these logs in prod by default or just ask whoever is trying to debug an issue to enable them in their cluster.

Reply:
Fixed